names.md

```diff
@@ -3,7 +3,7 @@ subl# This is a list of all the names of people working on this project
 # Format -   Full name - nickname (optional)
 Rucha Rathor
 Benjamin Chow - Ben
-Nitesh Goyal 
+Nitesh Goyal - green
```

Comment:
You based your current branch based off the color branch.  When you make a new branch you should base it off the master branch